scheduler-server/build.gradle

```diff
@@ -42,6 +43,11 @@ dependencies {
   testFixturesImplementation project(':merlin-driver')
 }
 
+processResources {
+  // Copy in scheduling DSL compiler static libraries when processing resources
+  from project(':scheduler-worker').projectDir.toPath().resolve(Path.of('scheduling-dsl-compiler', 'src', 'libs'))
```

Comment:
Why does `scheduling-dsl-compiler` live in the worker directory and not in the server directory? Asking because it is the opposite of how the merlin `constraints-dsl-compiler` is organized.

Reply:
Good question. So with `merlin-*` the server executes the Node DSL code. However, with `scheduler-*` the worker is the service that actually executes Node DSL code. The `scheduler-server` just needs access to static TS files but does not perform any TS execution.

Reply:
This is why we're restricting `scheduler-server`'s access to be limited to just `scheduling-dsl-compiler/src/libs/`.

Reply:
Might be a good idea to eventually move `constraints-dsl-compiler` and `scheduling-dsl-compiler` to the top-level Aerie repo directory since both are used by more than 1 subproject (ex: `constraints-dsl-compiler` is actually used within the `scheduling-dsl-compiler` through some Gradle copy-on-build business). But I think that should be done outside of this PR.